Nikita_Konkov_Application/Selenium_Basics/Selenium_Basics/EpamHowWeDoItOurWorkNavigationTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Selenium_Basics
+{
+    [TestFixture]
+    public class EpamHowWeDoItOurWorkNavigationTest
+    {
+        private IWebDriver webDriver = new ChromeDriver();
+        private string _epamHowWeDoItLink;
+        private string _epamOurWorkLink;
+
+        [SetUp]
+        public void Setup()
+        {
+            _epamHowWeDoItLink = "https://www.epam.com/how-we-do-it";
+            _epamOurWorkLink = "https://www.epam.com/our-work";
+        }
+
+        [Test]
+        public void CheckHowWeDoItAndOurWorkNavigation()
+        {
+            webDriver.Manage().Window.Maximize();
```

Comment:
in this case there should be no maximization :)

Reply:
Corrected it)